### PR TITLE
Update client types when editing participant

### PR DESCRIPTION
**Pull request by alex-yau-ttd**
https://github.com/IABTechLab/uid2-self-serve-portal/assets/137864392/e72f9d44-84d8-4b6d-a21e-94757a8fd20c

### CODE REVIEW

File: src/api/routers/participants/participantsCreation.ts [hunk 1]
```diff
@@ -3,32 +3,29 @@ import { z } from 'zod';
...
+  Participant, ParticipantStatus
 } from '../../entities/Participant';
-import { ParticipantType } from '../../entities/ParticipantType';
 import { User, UserCreationPartial } from '../../entities/User';
 import { getKcAdminClient } from '../../keycloakAdminClient';
 import { addSite, getSiteList, setSiteClientTypes } from '../../services/adminServiceClient';
 import {
   mapClientTypesToAdminEnums,
-  SiteCreationRequest,
+  SiteCreationRequest
```
Comment (ashleysmithTTD) [inline src/api/routers/participants/participantsCreation.ts:13]: I think lint-wise, this should have a comma like in the original - it gets auto corrected to a comma when I save it on my machine

File: src/api/routers/participants/participantsCreation.ts [hunk 2]
```diff
@@ -3,32 +3,29 @@ import { z } from 'zod';
...
 } from '../../services/adminServiceHelpers';
 import {
   insertAddParticipantAuditTrail,
-  updateAuditTrailToProceed,
+  updateAuditTrailToProceed
```
Comment (ashleysmithTTD) [inline src/api/routers/participants/participantsCreation.ts:17]: Same lint comment here

File: src/api/routers/participants/participantsCreation.ts [hunk 3]
```diff
@@ -3,32 +3,29 @@ import { z } from 'zod';
...
 } from '../../services/auditTrailService';
 import {
   assignClientRoleToUser,
   createNewUser,
-  sendInviteEmail,
+  sendInviteEmail
```
Comment (ashleysmithTTD) [inline src/api/routers/participants/participantsCreation.ts:22]: same lint comment here

File: src/api/routers/participants/participantsCreation.ts [hunk 4]
```diff
@@ -3,32 +3,29 @@ import { z } from 'zod';
...
 } from '../../services/kcUsersService';
-import { ParticipantRequest } from '../../services/participantsService';
+import { getParticipantTypesByIds, ParticipantRequest } from '../../services/participantsService';
 import { findUserByEmail } from '../../services/usersService';
 import {
   ParticipantCreationAndApprovalPartial,
-  ParticipantCreationRequest,
+  ParticipantCreationRequest
```
Comment (ashleysmithTTD) [inline src/api/routers/participants/participantsCreation.ts:28]: same lint here

File: src/api/services/participantsService.ts [hunk 5]
```diff
@@ -7,14 +7,13 @@ import {
   Participant,
   ParticipantApprovalPartial,
   ParticipantCreationPartial,
-  ParticipantDTO,
-  ParticipantStatus,
+  ParticipantDTO, ParticipantStatus
```
Comment (ashleysmithTTD) [inline src/api/services/participantsService.ts:10]: I think `ParticipantStatus` should be on its own line

**Review by ashleysmithTTD — APPROVED** [review]

File: src/api/routers/participants/participantsCreation.ts (hunk 1 above)
Comment (alex-yau-ttd) [reply src/api/routers/participants/participantsCreation.ts]: nice pickup. I think my auto-formatting broke - restarting VS Code seemed to fix them all.

File: .prettierrc [hunk 6]
```diff
@@ -6,6 +6,7 @@
   "singleQuote": true,
   "jsxSingleQuote": true,
   "endOfLine": "lf",
+  "trailingComma": "es5",
```
Comment (alex-yau-ttd) [inline .prettierrc:9]: Despite being the default, this was not being respected in VS Code.

**Pull request merged**